frontend/src/components/DeploymentDialog.vue

```diff
@@ -62,17 +62,25 @@ export default class DeploymentDialog extends Vue {
     }
 
     onStart () {
-        this.output.push('Deployment started', "Ignore first '[ERROR]:' line - this is harmless Ansible bug. :o)");
+        this.pushAutoScroll('Deployment started', "Ignore first '[ERROR]:' line - this is harmless Ansible bug. :o)");
```

Comment:
Unfortunately, our force-push ping pong resulted in stray commit `5fdbd51d5a4ec563486ab94d6c9a378002258b36` on your branch. 

This commit has been pushed to master prematurely and I removed it. You probably rebased on top of it. Pls remove it by `git rebase -i HEAD~2` and simply drop it.

I need to stop doing force-pushes while having collaborators. Sorry for that trouble.